src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -49,7 +49,7 @@ public static class CLIOptions
             ArgumentName = "--timeoutMS",
             ArgumentShortName = "-t <ms>",
             ArgumentDescription = "When passed, a logfile will be created for this mutationtest run on trace level",
-            DefaultValue = 2000,
+            DefaultValue = 30000,
```

Comment:
Default value has changed, please update the readme accordingly:
https://github.com/stryker-mutator/stryker-net/blame/master/src/Stryker.CLI/README.md#L36

Reply:
Ah good point